keyboards/aplusx_pulsar/xboard_qs/xboard_qs.c

```diff
@@ -0,0 +1,255 @@
+/* Copyright 2024 Aplusx Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
```

Comment:
```suggestion
```
Only need one or the other, and in this case, the explicitly named version is probably best, here.

Reply:
I got it. thank you